networkx/utils/backends.py

```diff
@@ -107,6 +92,65 @@
 Note that the ``backend_name`` is e.g. ``parallel``, the package installed
 is ``nx-parallel``, and we use ``nx_parallel`` while importing the package.
 
+.. _introspect:
+
+Introspection
+-------------
+Introspection techniques aim to demystify dispatching and conversion behaviors.
+They should help answer questions such as:
+
+- What happened (and why)?
+- What *will* happen (and why)?
+- Which backends implement a function?
+- What are the backend-specific arguments?
+- Where was time spent (including conversions)?
+- What is in the cache and how much memory is it using?
+
+We strive to make NetworkX backends easy to use, and we want them to "just work"
+for most users. Nevertheless, NetworkX dispatching should be transparent to allow
+for greater understanding, debug-ability, and customization. After all, NetworkX
+dispatching is extremely flexible and can support advanced workflows with multiple
+backends and fine-tuned configuration. It can support your needs as they evolve.
+
+Introspection capabilities are currently limited, but we are working to improve them.
+
+The primary way to see what the dispatch machinery is doing is by enabling logging.
+This can help you verify that the backend you specified is being used.
+You can enable networkx's backend logger to print to ``sys.stderr`` like this::
+
+    import logging
+    nxl = logging.getLogger("networkx")
+    nxl.addHandler(logging.StreamHandler())
+    nxl.setLevel(logging.DEBUG)
+
+And you can disable it by running this::
+
+    nxl.setLevel(logging.CRITICAL)
+
+Refer `this <https://docs.python.org/3/library/logging.html>`_ to learn more about
+the logging facilities in Python.
+
+You can see which currently installed backends implement a function by looking
+the ``.backends`` attribute, which is a set of backend names::
+
+    >>> nx.betweenness_centrality.backends  # doctest: +SKIP
+    {'parallel'}
```

Comment:
I would explicitly mention one more time that this (and the function docstring below) is an example, i.e. introduce it with "`For example,::`". (I got confused on first reading why I was getting `set()` locally.)

Alternatively, maybe emphasize "currently installed" somehow?

Reply:
Thanks for the suggestions! Is it more clear now?

Reply:
IMO this is better, yes. Thanks!